### DISCUSSION TITLE
is  it  running correct?

### DISCUSSION THREAD

**Discussion by viponedream** [swar/Swar-Chia-Plot-Manager#373] [Q&A]
i have two jobs, each max_concurrent is 3,  but after 2 plots finished  ( 6 hours later),  i found there are 7 active jobs shows in view.

GLOBAL：
  max_concurrent: 7

- name: AMD-SSD-1
    max_concurrent: 3
   ...

- name: AMD-SSD-2
   max_concurrent: 3
   ...

**Comment by swar** [comment]
This is not an issue. Please use GitHub Discussions instead. Or the Discord, Or Keybase.
